staging/src/k8s.io/apiserver/pkg/audit/types.go

```diff
@@ -44,3 +44,8 @@ type Backend interface {
 	// Returns the backend PluginName.
 	String() string
 }
+
+type Annotation struct {
```

Comment:
this type is at the wrong spot. It belongs into the admission code. For audit any annotation is the same.

Reply:
I agree. I first put it under `k8s.io/apiserver/pkg/admission` and let `k8s.io/apiserver/pkg/audit` imports it. It turns out to create `k8s.io/apiserver/pkg/admission` -> `k8s.io/apiserver/pkg/audit` -> `k8s.io/apiserver/pkg/admission` dependency cycle

I will try other places under admission (something like `.../admission/util`?)